extensions/amp-list/0.1/amp-list.js

```diff
@@ -771,18 +797,12 @@ export class AmpList extends AMP.BaseElement {
    * @private
    */
   moveButtonsToBottom_(container) {
-    if (this.loadMoreButton_) {
```

Comment:
Are these always defined now? `appendChild` will throw if it's passed a `null`.

Reply:
It does look better without the `if (exists) { exists.doStuff(); }` checks

Reply:
Yes. So long as we have `load-more`, we will always provide defaults for everything except load-end at `buildCallback`.